XdmodAID.md

```diff
@@ -0,0 +1,853 @@
+the easiest and fastest way to point out to the customer why a charge appeared on their invoice.
+
+It is quite painful to explain things to customers without an easy to digest transaction
+log.
+
+##advantages to using xdmod
+```
+  1. community of users
+  2. Seems to have a reeasonable database structure
+  3. does more of what we want to do than cloud forms did
+  4. cloud be a unified platform for collecting and storing metrics on OpenShfit, OpenStack
+     and HPC clusters
+```
+##disavantages to using xdmod
+```
+  1. Not turnkey for cloud services
+  2. No built in support for openshift
+  3. OpenStack support is for a version that is no longer maintained
+  4. The xdmod team does not seem responsive to their ticketing system
+  5. Xdmod is a very large package and it doesn't have much flexibility.
+```
+Like any legacy system, there are parts of xdmod that are done well and parts
+that are not.  I have the impression that the xdmod team does not view cloud
+support as a priority as they themselves don't deploy any current cloud
+service.
+
+It would not suprise me if we decide to drop xdmod only to develop
+Furthermore, it would not suprise me if there was one application for OpenShift
+and another for OpenStack
+
+#Dockerfiles
+
+Here are the dockerfiles to the project:
+```
+  1. Dockerfile.moc-xdmod
+  2. Dcokerfile.moc-xdmod-dev
+  3. Dockerfile.xdmod-openstack
+  4. Dockerfile.docker-test
+```
+
+#Dockerfile.moc-xdmod
+Initailly I copied over the docker files from the xdmod github repository as I
+figured there was the distinct possiblity that some restructuring would be
+necessary.  However, I also wanted to see if the original docker files could be used
+as is, as this would potentally make upgrading easier as we would be using what
+their process developed.
+
+I had put some changes in to dockerfile to support xdmod-openstack and similar
+changes were done to support xdmod-shift.  Along the way, in order to get PRs
+approved, we decided that we needed to optimize the dockerfiles.
+
+Subsequently, I have moved the changes that I made to the original dockerfile
+to a different one based on python 3.11 dockerfile.  Likewise, the xdmod-openshift
+changes could be isolated.
+
+Currently, I am uncertain if the original dockerfile would still work for us, or if we
+could drop in the xdmod team's dockerfile.  I had planned to do this after
+moving xdmod-openshift into it's own dockerfile.
+
+#Basic setup
+
+Installation and initial configuration is handled in the Dockerfile.moc-xdmod
+
+##realms and resources
+A realm is the main reporting area.  Here are the list of realms
+```
+    job     - handles basic openshift
+    cloud   - handles openstack
+    supremm - should give some performance metrics for openshift
+```
+A resource can be a part of the realms, as I haven't really started to work on the
+storage resources (possibly part of a storage realm), I have confirmed this yet
```

Comment:
this sentence needs a look, I am not sure if you meant to say that you have not confirmed this yet or that you have. "I have confirmed this by actually setting them up." or "I have not confirmed this yet by actually setting them up."

Reply:
this still needs attention.